#right encoder value
int32 right_encoder

#left encoder value
int32 left_encoder

#interval of two times receive
float64 interval

#wheel encoder status bool
bool encoderWrong
